internal/common/baseclient/baseSpeaker.go

```diff
@@ -33,12 +33,20 @@ func (s *BaseSpeaker) DecideAnnouncement(ruleId string, result bool) (string, bo
 }
 
 // CallJudgeElection is called by the legislature to decide on power-transfer
-func (s *BaseSpeaker) CallJudgeElection(turnsInPower int, allIslands []shared.ClientID) shared.ElectionSettings {
+func (s *BaseSpeaker) CallJudgeElection(monitoring shared.MonitorResult, turnsInPower int, allIslands []shared.ClientID) shared.ElectionSettings {
+	// example implementation calls an election if monitoring was performed and the result was negative
+	// or if the number of turnsInPower exceeds 3
 	var electionsettings = shared.ElectionSettings{
 		VotingMethod:  shared.Plurality,
 		IslandsToVote: allIslands,
 		HoldElection:  true,
 	}
+	if monitoring.Performed && monitoring.Result {
+		electionsettings.HoldElection = false
+	}
+	if turnsInPower >= 2 {
+		electionsettings.HoldElection = false
+	}
```

Comment:
contradicting comment? 
electionsettings.HoldElection = true?

Reply:
Same for the other base clients.

Reply:
good spot! yes, this was completely backwards haha. sorted it now!